spectractor/fit/fit_multispectra.py

```diff
@@ -533,18 +537,17 @@ def simulate(self, aerosols, angstrom_exponent_log10, ozone, pwv, reso, *A1s):
         >>> assert np.sum(w.amplitude_params) > 0
 
         """
-        if self.fit_angstrom_exponent:
-            angstrom_exponent = 10 ** angstrom_exponent_log10
-        else:
+        if not self.fit_angstrom_exponent:
             angstrom_exponent = None
         # linear regression for the instrumental transmission parameters T
         # first: force the grey terms to have an average of 1
         deltas = np.array(A1s)[:self.nspectra]
         A1s = np.array(A1s)[self.nspectra:]
-        if A1s.size > 1:
-            m = 1
-            A1s[0] = m * A1s.size - np.sum(A1s[1:])
-            self.params.values[self.A1_first_index] = A1s[0]
+        # if A1s.size > 1:
+        #     m = 1
+        #     A1s[0] = m * A1s.size - np.sum(A1s[1:])
+        #     self.params.values[self.A1_first_index] = A1s[0]
```

Comment:
As always, I'd encourage delete of keep, but not leaving commented code, but it's up to you, of course.

Reply:
Yes, but this one I will keep it because not sure of my code. `fit_multispectra.py` is still under construction and test